datasets/No_Accounts_08012022/data.sql

```diff
@@ -524,7 +524,7 @@ INSERT INTO "EASY_Widget__c" VALUES(72,'Active','','<p><strong style="color: rgb
-INSERT INTO "EASY_Widget__c" VALUES(76,'Active','','<h3 style="text-align: center;"><strong style="font-size: 20px; color: rgb(109, 54, 150);">Welcome to the University of St. Thomas application!</strong></h3><p style="text-align: center;"><br></p><h3 style="text-align: center;"><a href="https://uofstthomasmn--edadevuat.sandbox.my.site.com/admissions/applicationrequirement?at=GSOE&amp;requirementId=a2c2h000000gL6oAAE" target="_blank"><img src="https://uofstthomasmn--edadevuat.sandbox.file.force.com/servlet/rtaImage?eid=a2L2h000000cMAg&amp;feoid=00N2h000002Bu9U&amp;refid=0EM2h000000IuzA" alt="User-added image"></img></a></h3>','','ApplicationPortal','Started App','','','','Graphic: GR Continue (GSOE)','','','','','Content Top','Central Daylight Time (America/Chicago)','Custom','7');
```

Comment:
This data set will be wiped out at some point and since widgets are user generated into data we will have to train said users to follow you accessible convention.

Reply:
Do they copy forward from old data sets or write fresh each time? The gist of it is just that the header hierarchy has to be correct. Would this be something I should talk to Mike and Farrah about?